appendix-works-consulted.md

```diff
@@ -99,6 +99,9 @@ Purposeful Open Source*, 28 October 2019, Open Tech Strategies,
 * Fogel, Karl. *Producing Open Source Software: How to Run a Successful Free 
 Software Project*, 14 Aug. 2020, [producingoss.com](https://producingoss.com). Accessed 1 Sept. 2021.
 
+* Foxman, Stephen M.  Sample Contract Clauses. https://www.eckertseamans.com/app/uploads/Steve-Foxman_Eckert-Seamans_DATA-PROTECTION-CLAUSES-101016-M1566466xA35AF.pdf).  
```

Comment:
Ah, I think we might need to be clearer about attributing the specific bit of material to Foxman, to implement the attribution as he requested?

Reply:
The attribution was clear, but he felt the earlier ordering made it seem like they'd also written the OSQA section, which isn't the case. And since we haven't found any better sample contract language, I removed the TODO and included a formal reference in Works Consulted.

Reply:
Gotcha.  What I'm getting at is just the question of whether that particular attribution needs to be right near (e.g., directly above) the attributed material, as well as in Works Consulted.  I think it does?  That is, I think that the kind of attribution they had in mind is "When the reader encounters our material, they should be able to see right then and there where it comes from."

Reply:
It does not need to be more clearly attributed. They approved the content I shared with them -- the existing public info on the "main" branch -- and only asked that the ordering be changed. We do reference that the sample contract is from Eckert Seamans in the text just above it, although I suppose it could be made even more clear. I'm sure they'd be fine with that change.

Reply:
Ah, okay -- you're clearly more on top of this than I am!  No change needed; sorry for pushing on something that you had completely solved already.